### PR TITLE
blog post for june community call

### PR DESCRIPTION
**Pull request by matthidinger**
Blog post is available here: https://adaptivecardsci.z5.web.core.windows.net/pr/4195/blog/2020/Community-Call-June/

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4195)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
@matthidinger while you are at it, can you update the Feedback link on the blog post site to redirect to the [New Bug Template](https://github.com/microsoft/AdaptiveCards/issues/new?assignees=&labels=Bug&template=bug_report.md&title=%5BPlatform%5D%5BBroad+Issue+Category+if+applicable+%28eg+Accessibility%29%5D+%5BBug+Title%5D) instead of the default blank one? Thanks

File: source/nodejs/adaptivecards-site/pages/_posts/2020/Community-Call-June.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Build 2020 and a look at the Roadmap
+------------------------------------
+
+Build 2020 was all-virtual this year, and we had the opportunity to share the latest and greatest coming to Adaptive Cards on the [YouTube session](https://aka.ms/m365sk134). The session focused on four areas, outlined below.
+
+### Adaptive Card Templating and Designer Updates
+
+Templating helps you save time by easily populating reusable card templates with data. [Release Candidate SDKs](https://docs.microsoft.com/en-us/adaptive-cards/templating/sdk) are now available for .NET and JavaScript, and we plan to make them Generally Available in the coming weeks. We also added rich data-binding support in the [Adaptive Card Designer](https://adaptivecards.io/designer) to make it even faster to design new cards based on your data payloads.
+
```
Comment (shalinijoshi19) [inline source/nodejs/adaptivecards-site/pages/_posts/2020/Community-Call-June.md:22]: [question] Are we referring to templating and data binding interchangeably at this point?

Comment (matthidinger) [reply source/nodejs/adaptivecards-site/pages/_posts/2020/Community-Call-June.md:22]: It's a fair question. In this context I'm using data-binding to refer to the act of binding to data, which is what the Designer helps do with the little flyout menu that lets you pick a property to bind to. Templating generally refers to overall feature area, and includes the ability to write expressions, hide objects with `$when`, etc. Eventually I hope we can release an Expression Builder UI into the Designer (Like Power Automate has)

**Pull request merged**